kiosko/templates/404.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:template-part {"slug":"header","tagName":"header","theme":"kiosko"} /-->
```

Comment:
This is coming from Create Block Theme plugin. I don't remember why the plugin adding this. 
I think it is a fix for the plugin?

CC: @matiasbenedetto

Reply:
Sorry, I am referring to this [commit](https://github.com/Automattic/themes/pull/6905/commits/47bfbf8f6ff76fb31a7c54adf85b1e0820982a0d)

<img width="657" alt="image" src="https://user-images.githubusercontent.com/1935113/231794867-2bc478fb-24fc-4722-8871-c9ceb2d87c68.png">